### DISCUSSION TITLE
Font Picker component

### DISCUSSION THREAD

**Discussion by ilomon10** [mantinedev/mantine#3630] [Q&A]
Any font picker component to select font like google fonts?
Or any suggestion any library to pick font interactively

**Comment by rtivital** [comment]
Use Select component

**Reply by ilomon10** [reply]
i mean the ready to use font picker with google font api, or any snippet?

**Reply by rtivital** [reply]
We do not have it

**Reply by ilomon10** [reply]
ok thanks 👍🏻
